src/components/Task/Drawer/Form/Form.tsx

```diff
@@ -58,6 +71,13 @@ const useStyles = makeStyles((theme: Theme) => ({
   title: {
     flexGrow: 1,
   },
+  removeButton: {
+    backgroundColor: '#d9534f',
+    color: '#fff',
+    '&:hover': {
+      backgroundColor: '#C9302F',
+    },
```

Comment:
Are these theme colors?

Reply:
```#fff``` is one so I switched to using it from the theme instead of hard coding it here. ```#d9534f``` was the color from the delete button on mpdx.org that I just copied over, but I'm going to instead use the error color from the theme. I'm also going to add ```#C9302F``` to the theme as the dark version of the error color.